dependencies.properties

```diff
@@ -282,27 +282,27 @@ org.slf4j\:slf4j-api=1.7.12
 org.slf4j\:slf4j-jdk14=1.7.12
 org.slf4j\:slf4j-log4j12=1.7.12
 org.spockframework\:spock-core=0.7-groovy-2.0
-org.springframework\:spring-aop=4.2.2.RELEASE
-org.springframework\:spring-aspects=4.2.2.RELEASE
-org.springframework\:spring-beans=4.2.2.RELEASE
-org.springframework\:spring-context=4.2.2.RELEASE
-org.springframework\:spring-context-support=4.2.2.RELEASE
-org.springframework\:spring-core=4.2.2.RELEASE
-org.springframework\:spring-expression=4.2.2.RELEASE
-org.springframework\:spring-instrument=4.2.2.RELEASE
-org.springframework\:spring-instrument-tomcat=4.2.2.RELEASE
-org.springframework\:spring-jdbc=4.2.2.RELEASE
-org.springframework\:spring-jms=4.2.2.RELEASE
+org.springframework\:spring-aop=4.2.3.RELEASE
+org.springframework\:spring-aspects=4.2.3.RELEASE
+org.springframework\:spring-beans=4.2.3.RELEASE
+org.springframework\:spring-context=4.2.3.RELEASE
+org.springframework\:spring-context-support=4.2.3.RELEASE
+org.springframework\:spring-core=4.2.3.RELEASE
+org.springframework\:spring-expression=4.2.3.RELEASE
+org.springframework\:spring-instrument=4.2.3.RELEASE
+org.springframework\:spring-instrument-tomcat=4.2.3.RELEASE
+org.springframework\:spring-jdbc=4.2.3.RELEASE
+org.springframework\:spring-jms=4.2.3.RELEASE
 org.springframework\:springloaded=1.2.4.RELEASE
-org.springframework\:spring-messaging=4.2.2.RELEASE
-org.springframework\:spring-orm=4.2.2.RELEASE
-org.springframework\:spring-oxm=4.2.2.RELEASE
-org.springframework\:spring-test=4.2.2.RELEASE
-org.springframework\:spring-tx=4.2.2.RELEASE
-org.springframework\:spring-web=4.2.2.RELEASE
-org.springframework\:spring-webmvc=4.2.2.RELEASE
-org.springframework\:spring-webmvc-portlet=4.2.2.RELEASE
-org.springframework\:spring-websocket=4.2.2.RELEASE
+org.springframework\:spring-messaging=4.2.3.RELEASE
```

Comment:
Is there a requirement to upgrade spring version to `4.2.3` or are we just updating to the latest release?